src/pages/login/Login.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { BASE_URL } from '../../helper/url'
+
+Login.propTypes = {
+  setToken: PropTypes.func.isRequired,
+}
+
+export default function Login({ setToken }) {
+  const [username, setUserName] = useState()
+  const [error, setError] = useState(null)
+
+  async function loginUser(credentials) {
+    return fetch(BASE_URL + '/users/' + username + '/claim', {
```

Comment:
When writing it like this, you are using the state and not the argument you pass the function, which I would recommend doing. Then the function is not dependent on the state and you could potentially move it in a folder `lib` or `services`.
```suggestion
    return fetch(BASE_URL + '/users/' + credentials + '/claim', {
```